### PR TITLE
Fd23

### PR DESCRIPTION
**Pull request by Doomsdayrs**

### CODE REVIEW

File: io.itch.itch.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 app-id: io.itch.itch
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '23.08'
 sdk: org.freedesktop.Sdk
 base: org.winehq.Wine
 base-version: stable-22.08
```
Comment (AsciiWolf) [inline io.itch.itch.yaml:6]: @Doomsdayrs Why did you keep base (Wine) at 22.08?

**Comment by AsciiWolf** [comment]
By the way, README.md should also be updated: https://github.com/flathub/io.itch.itch#development

File: io.itch.itch.yaml (hunk 1 above)
Comment (Doomsdayrs) [reply io.itch.itch.yaml:6]: Because they did not update wine at that point

Comment (Doomsdayrs) [reply io.itch.itch.yaml:6]: Fixed with #12

**Pull request merged**